implementation/src/test/java/io/smallrye/mutiny/operators/UniAndTest.java

```diff
@@ -20,30 +20,34 @@ public void testWithTwoSimpleUnis() {
         Uni<Integer> uni = Uni.createFrom().item(1);
         Uni<Integer> uni2 = Uni.createFrom().item(2);
 
-        UniAssertSubscriber<Tuple2<Integer, Integer>> subscriber = uni.and().uni(uni2).asTuple().subscribe()
+        UniAssertSubscriber<Tuple2<Integer, Integer>> subscriber = Uni.combine().all().unis(uni, uni2).asTuple().subscribe()
```

Comment:
I would have kept some usage of the deprecated API to be sure we don't break it in the future (before the official removal).

Reply:
Typically, I would have kept this class unchanged, just mentioning it test a deprecated API until we remove it.

Reply:
Perhaps duplicating all tests is too much, and having these changes prepares for the day we remove the deprecated APIs.

What if I duplicate a limited set of previous tests? (but not all or reverting the corresponding commit)

Reply:
Yes, it's actually what I meant, I generally keep one or two tests.

Reply:
Added a few basic ones in https://github.com/smallrye/smallrye-mutiny/pull/194/commits/5f6923d63d8d339ddf2e76a82ba0b3e82b7ac9fd, is that ok?

Reply:
Yes! Thanks!